mainnet-1/specs/stargaze.json

```diff
@@ -0,0 +1,787 @@
+              },
+              {
+                "name": "osmosis.tokenfactory.v1beta1.Query/DenomsFromCreator",
+                "block_parsing": {
+                  "parser_arg": [
+                    "latest"
+                  ],
+                  "parser_func": "DEFAULT"
+                },
+                "compute_units": 10,
+                "enabled": true,
+                "category": {
+                  "deterministic": true,
+                  "local": false,
+                  "subscription": false,
+                  "stateful": 0
+                },
+                "extra_compute_units": 0
+              }
+            ],
+            "headers": [],
+            "inheritance_apis": [],
+            "parse_directives": [],
+            "verifications": [
+              {
+                "name": "chain-id",
+                "values": [
+                  {
+                    "expected_value": "stargaze-1"
+                  }
+                ]
+              },
+              {
+                "name": "pruning",
+                "values": [
+                  {
+                    "latest_distance": 14900
+                  },
+                  {
+                    "extension": "archive",
+                    "expected_value": "1"
+                  }
+                ]
+              }
+            ],
+            "extensions": [
+              {
+                "name": "archive",
+                "cu_multiplier": 5,
+                "rule": {
+                  "block": 14600
+                }
+              }
+            ]
+          },
+          {
+            "enabled": true,
+            "collection_data": {
+              "api_interface": "tendermintrpc",
+              "internal_path": "",
+              "type": "",
+              "add_on": ""
+            },
+            "apis": [],
+            "headers": [],
+            "inheritance_apis": [],
+            "parse_directives": [],
+            "verifications": [
+              {
+                "name": "chain-id",
+                "values": [
+                  {
+                    "expected_value": "stargaze-1"
+                  }
+                ]
+              },
+              {
+                "name": "pruning",
+                "values": [
+                  {
+                    "latest_distance": 14900
+                  },
+                  {
+                    "extension": "archive",
+                    "expected_value": "1"
+                  }
+                ]
+              }
+            ],
+            "extensions": [
+              {
+                "name": "archive",
+                "cu_multiplier": 5,
+                "rule": {
+                  "block": 14600
+                }
+              }
+            ]
+          }
+        ],
+        "contributor": [
+          "lava@1jnsszemt05kcxd4pf3uzewl9xqtzfpfg65960w"
+        ],
+        "contributor_percentage": "0.025",
+        "identity": "7C0AE2E5F08269E6"
```

Comment:
This should be the logo of Axelar, if you don't have it, you can leave empty (= removed)

Reply:
The field value has been updated [f945313](https://github.com/lavanet/lava-config/pull/24/commits/f945313bd76273364a1dd62f8aa8357432ed92d8)